Libplanet.Explorer/GraphTypes/PolicyType.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Linq;
+using GraphQL.Types;
+using Libplanet.Action;
+using Libplanet.Blockchain.Policies;
+
+namespace Libplanet.Explorer.GraphTypes;
+
+public class PolicyType<T> : ObjectGraphType<IBlockPolicy<T>>
```

Comment:
"Policy" seems to mean a little bit broad. how about `BlockPolicy` (from `IBlockPolicy`), or `ChainPolicy` (newly introduced, but more specific).

Reply:
Renamed in 3379ec73078f9994caa7a3c27798b13474cfca9a.